adoptium-updater-parent/adoptium-api-v3-updater/src/test/kotlin/net/adoptium/api/AdoptReleaseMapperTest.kt

```diff
@@ -231,10 +138,78 @@ class AdoptReleaseMapperTest : BaseTest() {
             assertEquals(2, release.result!![1].binaries.size)
 
             assertEquals(1, release.result!![0].download_count)
-            assertEquals(3, release.result!![1].download_count)
+            assertEquals(2, release.result!![1].download_count)
         }
     }
 
+    private fun mockClient() = object : UpdaterHtmlClient {
+        override suspend fun get(url: String): String {
+            return getMetadata(url)
+        }
+
+        fun getMetadata(url: String): String {
+            val opt = UUID.randomUUID()
+
+            val build = try {
+                Integer.parseInt(url)
+            } catch (ignore: Exception) {
+                1
```

Comment:
not sure what the 1 does here?

Reply:
in the case that we cannot parse the number, defaults it to 1